common/djangoapps/entitlements/models.py

```diff
@@ -1,8 +1,105 @@
 import uuid as uuid_tools
+from datetime import datetime, timedelta
 
+import pytz
 from django.conf import settings
+from django.contrib.sites.models import Site
 from django.db import models
+
 from model_utils.models import TimeStampedModel
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+
+
+class CourseEntitlementPolicy(models.Model):
+    """
+    Represents the Entitlement's policy for expiration, refunds, and regaining a used certificate
+    """
+
+    DEFAULT_EXPIRATION_PERIOD_DAYS = 450
+    DEFAULT_REFUND_PERIOD_DAYS = 60
+    DEFAULT_REGAIN_PERIOD_DAYS = 14
+
+    expiration_period_days = models.IntegerField(
+        default=DEFAULT_EXPIRATION_PERIOD_DAYS,
+        help_text="Number of days from when an entitlement is created until when it is expired.",
+        null=False
+    )
+    refund_period_days = models.IntegerField(
+        default=DEFAULT_REFUND_PERIOD_DAYS,
+        help_text="Number of days from when an entitlement is created until when it is no longer refundable",
```

Comment:
Nit: it might be good to specify that this is the **max** number of days until you can no longer get a refund. The refund period may actually end sooner if the entitlement is spent.